plugins/become/enable.py

```diff
@@ -6,8 +6,9 @@
 __metaclass__ = type
 
 DOCUMENTATION = """
-author: Ansible Networking Team
-become: enable
+author:
+ - Ansible Networking Team (@network-team)
```

Comment:
@network-team doesn't seem to be an actual GitHub id.

Reply:
ack